apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/TraceContextForLogEntry.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2020 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.Diagnostics.Common;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    /// Represents a trace context (Trace ID and Span ID)
+    /// for a log entry.
+    /// These values can be attached to a log entry to establish the
+    /// relation of it and a trace.
+    /// </summary>
+    public sealed class TraceContextForLogEntry
+    {
+        /// <summary>
+        /// The trace ID to associate the log entry with.
+        /// Must not be null.
+        /// </summary>
+        public string TraceId { get; }
+
+        /// <summary>
+        /// The span ID to associate the log entry with.
+        /// Or null if there's currently no open span.
+        /// </summary>
+        /// <remarks>
+        /// This is the same format that the Trace API v2 uses: a 16-character hexadecimal
```

Comment:
Presumably that's only if that's what was passed to the constructor though - if I call `new TraceContextForLogEntry("trace-id", "this-is-not-hex")` then it's not going to complain.

Reply:
Yes, I'm modifying the comments, as in, "it should be in this format, but we don't check"